src/vm/class.h

```diff
@@ -1936,7 +1946,8 @@ class EEClass // DO NOT CREATE A NEW EEClass USING NEW!
         // unused                              = 0x00010000,
         VMFLAG_NO_GUID                         = 0x00020000,
         VMFLAG_HASNONPUBLICFIELDS              = 0x00040000,
-        // unused                              = 0x00080000,
+
+        VMFLAG_HAS_FLOATING_POINT_NUMBER       = 0x00080000, // Indicates whether floating point number fields exist
```

Comment:
If we have a double or single field, with this bit,  we can avoid a dfs. Otherwise, we need to do dfs for double or single field as what we did for Equals and GetHashCode.